assets/src/apps/authoring/components/PropertyEditor/custom/CustomDropdownTemplate.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useState } from 'react';
+import { SequenceDropdown } from './SequenceDropdown';
+import {
+  findInHierarchy,
+  getHierarchy,
+  SequenceEntry,
+  SequenceEntryChild,
+} from 'apps/delivery/store/features/groups/actions/sequence';
+import { selectSequence } from 'apps/delivery/store/features/groups/selectors/deck';
+import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
+import { UiSchema } from '@rjsf/core';
+
+interface CustomDropdownProps {
+  id: string;
+  label: string;
+  uiSchema: UiSchema;
+  description: string;
+  properties: object;
+  value: string;
+  onChange: (value: string) => void;
+}
+const CustomDropdownTemplate: React.FC<CustomDropdownProps> = (props) => {
```

Comment:
since this isn't just *any* custom dropdown, it's specifically the Sequence / Screen picker dropdown, let's name it that way please. `ScreenSelectDropDown` or something like that